scripts/core/editor3/components/embeds/EmbedButton.jsx

```diff
@@ -44,8 +44,9 @@ export class EmbedButton extends Component {
         const {dialogOpen} = this.state;
 
         return (
-            <div className="Editor3-styleButton">
-                <span onClick={this.showInput}><i className="icon-code" /></span>
+            <div className="Editor3-styleButton Editor3-styleButton--tooltip-on-span">
+                <span onClick={this.showInput} data-flow={'down'} data-sd-tooltip="Embed">
+                    <i className="icon-code" /></span>
```

Comment:
Would you mind putting the `</span>` on the next line and aligning it with its opening tag?